src/Compilers/VisualBasic/Portable/VBResources.resx

```diff
@@ -5198,6 +5198,8 @@
 -errorreport:&lt;string&gt;             Specifies how to handle internal compiler
                                   errors; must be prompt, send, none, or queue
                                   (default).
+-generatedfilesout:&lt;dir&gt;            Place files generated during compilation in the
```

Comment:
```suggestion
-generatedfilesout:&lt;dir&gt;          Place files generated during compilation in the
```

Otherwise, the output is misaligned:

<img width="601" alt="image" src="https://user-images.githubusercontent.com/3669664/215713242-efeaade8-3a00-4ff8-b10a-41e55c70ded3.png">